md/resource-management.md

```diff
@@ -20,19 +25,19 @@ resources=[GET|living/application,GET|living/application-page,GET|living/applica
 contentType=layout
 ```
 
-* The `resources` directory contains all the public files of your resource (for example `index.md`, images, Javascript files, CSS files). 
+* The `resources` directory contains all the public files of your resource (for example `index.md`, images, Javascript files, CSS files).  
 
 * The zip archive must contain at least one of the following index files:
    * An `index.html` file in the `resources` directory
    * An `Index.groovy` class at the root of the archive with, optionally, libraries
 
-If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.
+If a resource contains both `Index.groovy` and `index.html`, the Groovy class takes precedence.  
 
-## Authorization permissions for resources
+## Permissions for resources
 
 If you are using [REST API authorization](rest-api-authorization.md) and your resource needs the Web REST API, 
 you need to specify the permissions that a user needs to have in order to access those API. 
-These permissions are defined in the `page.properties` file. If your resources is a page written in Groovy and uses the Bonita Engine Java APIs, you do not need to specify permissions.
+These permissions are defined in the `page.properties` file. If your resource is a page written in Groovy and uses the Bonita Engine Java APIs, you do not need to specify permissions.
 For each REST resource accessed in other resources, specify the authorization needed for each method used. 
 You can find examples of the default resources in [`resources-permissions-mapping.properties`](BonitaBPM_platform_setup.md).
 
```

Comment:
```suggestion

When a user is given access to a page/layout because he is in a profile that contains it or that is mapped to an application that contains the page/layout, then this user it granted upon login the permissions associated to these REST resources (see [REST API authorizations](rest-api-authorization.md) for more details).  

```